docs_sources/erp-eml-epa-notes.adoc

```diff
@@ -0,0 +1,130 @@
+
+=== Erzeugen von Medications für Rezepturen und Kombipackungen
+
+Die Darstellung von Rezepturen und Kombipackungen in der ePA unterscheidet sich zum Profil KBV_PR_ERP_Medication_Compounding.
+
+In KBV_PR_ERP_Medication_Compounding sind alle Bestandteile einer Rezeptur in _einer_ Medication als PZN Codes unter .ingredient aufgelistet. Dies bietet Vorteile für die Datensparsamkeit und reduziert die Komplexität der Daten.
+
+Für die Abgabe von Rezepturen hat das technisch u.A. den Nachteil, dass nicht für jeden PZN-Bestandteil der Rezeptur eine Angabe zur Chargeninformation angegeben werden kann.
+
+Die ePA bietet folgende Profile mit Besonderheiten:
+
+[cols="1,2"]
+|===
+|Profil|Besonderheit
+
+|link:https://simplifier.net/epa-medication/epamedication[EPA Medication] a|
+* Generisches Medication Profil
+* GEM_ERP_PR_Medication ist hiervon abgeleitet
+* Kann Kindelemente als .contained enthalten
+|link:https://simplifier.net/epa-medication/epamedicationpzningredient[EPA Medication Ingredient] a|
+* Medication zur Angabe eines Bestandteils einer Rezeptur
+* Kann keine weiteren Medications unter .ingredient.itemReference enthalten
+|link:https://simplifier.net/epa-medication/epamedicationpharmaceuticalproduct[EPA Pharmaceutical Product Medication] a|
+* Medication zur Angabe eines pharmazeutischen Produkts
+* Enthält den Produktschlüssel unter .code.coding:product-key
+
+|===
+
+==== Rezepturen
+
+Für Rezepturen wird genau eine EPA Medication erzeugt. Diese Medication enthält unter .extension:type den Wert "Extemporaneous preparation (product)". Damit ist diese Medication als Rezeptur gekennzeichnet.
+
+Für jeden Bestandteil der Rezeptur wird  eine EPA Medication Ingredient erzeugt und als .contained hinzugefügt. Die EPA Medication Ingredient Objekte können keine weiteren Medications enthalten, sondern nur die Angaben zu einem Bestandteil einer Rezeptur.
+
+Schematisch stellt sich eine Rezeptur wie folgt dar:
+
+image:medication-rezeptur-schematics.png[width=50%]
+
+.Beispiel eines Rezeptur FHIR-Datensatzes (Klicken zum Ausklappen)
+[%collapsible]
+//TODO: Change to real example from Examples Repository
+====
+[source,xml]
+----
+include::../resources/temp/Medication-Medication-Rezeptur.xml[]
+----
+====
+
+==== Kombipackung
+
+Das Erzeugen von Kombipackungen geschieht analog zur Rezeptur. Statt der Ingredient Profile wird das Profil EPA Medication Pharmaceutical Product für die Angabe der Bestandteile einer Kombipackung genutzt.
+
+.Beispiel eines Kombipackung FHIR-Datensatzes (Klicken zum Ausklappen)
+[%collapsible]
+//TODO: Change to real example from Examples Repository
+====
+[source,xml]
+----
+include::../resources/temp/Medication-Medication-Kombipackung.xml[]
+----
+====
+
+== Konzept für die Abgabe
+
+Zusammenfassend lässt sich die Abgabe von E-Rezepten ab dem 15.01.2025 mit den Workflow-Profilen der Version 1.4 wie folgt schematisch darstellen:
+
+image:dispense-1.4-schema.png[width=100%]
```

Comment:
Kann man in diesem Bild "andeuten" das die Operation $provide-otc am MedicationService auch den identischen Aufbau haben wird. Vielleicht gestrichelte Linien, oder in einer "epa-Wolke". Oder beides mit dem Zusatz "geplant für ePA 3.1"